caffe2/core/operator.h

```diff
@@ -151,6 +149,11 @@ class CAFFE2_API OperatorBase : public Observable<OperatorBase> {
     return outputs_.at(idx);
   }
 
+  // Check whether output j is an alias of input i
```

Comment:
Be careful. Because this only checks for a very specific kind of aliasing, which is if the blob pointers are the same or not, and not if the blobs point to the same tensor, or if the tensor pointers point to the same TensorImpl, or if the storages alias. Worth pointing this out in the comment.